custom/icds_reports/static/js/directives/map-or-sector-view/map-or-sector-view.directive.js

```diff
@@ -1,9 +1,11 @@
 /* global d3 */
 
-function MapOrSectorController() {
+function MapOrSectorController($scope, $location, storageService, locationsService) {
```

Comment:
```$scope``` is not used in the code. Please remove it.

Reply:
removed